src/components/EditCoursePage/index.jsx

```diff
@@ -116,6 +137,7 @@ class EditCoursePage extends React.Component {
     const entitlement = entitlements && entitlements[0];
     const mode = entitlement && entitlement.mode;
     const price = entitlement && entitlement.price;
+    const courseRuns = (course_runs && course_runs.length > 0) ? course_runs : [];
```

Comment:
You shouldn't need the check that length is greater than 0 if you're defaulting to `[]` right?